sherpa/astro/xspec/tests/test_xspec.py

```diff
@@ -735,6 +737,21 @@ def test_evaluate_xspec_model(modelcls):
 
     Convolution models are skipped (easier to filter out here given the
     current design).
+
+    This test checks two independent things:
+    1. The model can be evaluated with the default parameters
+       and the results agree and on energy and wavelength grids.
```

Comment:
```suggestion
       and the results agree using energy and wavelength grids.
```

Not sure what you meant `and on` to be here?